appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmRequestUtils.java

```diff
@@ -38,6 +38,9 @@ public static boolean isTrustedRemoteAddr(boolean isDevMode, String remoteAddr)
       return true;
     } else if (remoteAddr.startsWith(LINK_LOCAL_IP_NETWORK)) {
       return true;
+    } else if (remoteAddr.equals("127.0.0.2")) {
```

Comment:
Curious about the semantics required here. [RFC5735](https://tools.ietf.org/html/rfc5735) says:
>127.0.0.0/8 - This block is assigned for use as the Internet host loopback address.  A datagram sent by a higher-level protocol to an address anywhere within this block loops back inside the host.  This    is ordinarily implemented using only 127.0.0.1/32 for loopback.  As   described in [RFC1122], Section 3.2.1.3, addresses within the entire   127.0.0.0/8 block do not legitimately appear on any network anywhere.

Which means we should either trust all "127." addresses in the block, or only the "ordinary implementation" of "127.0.0.1" but treating just 127.0.0.2 as special seems odd.